src/components/styles/App.css

```diff
@@ -0,0 +1,5 @@
+#app-container {
+  width: 700px;
+  display: flex;
+  flex-direction: column;
+}
```

Comment:
## [optional]
if you center it, things will look more presentable ans symmetrical
` margin: auto `